export/index.ts

```diff
@@ -81,6 +82,15 @@ class NeonClient extends Client {
       this.password === null
     ) throw new Error(`No database host or connection string was set, and key parameters have default values (host: localhost, user: ${defaultUser}, db: ${defaultUser}, password: null). Is an environment variable missing? Alternatively, if you intended to connect with these parameters, please set the host to 'localhost' explicitly.`);
 
+    if (this.host !== 'localhost') {
+      // @ts-ignore -- TS doesn't know about this.connectionParameters
+      if (typeof this.connectionParameters.options !== 'string') {
+        // @ts-ignore
+        this.connectionParameters.options = '';
+      }
+      // @ts-ignore
+      this.connectionParameters.options += 'project=' + this.host.split('.')[0];
```

Comment:
(1) Currently proxy uses the same logic for both: `project` and `endpoint`, but sure, let's use `endpoint` since the project is a legacy option.

(2) Good point, let me fix this.